### PR TITLE
add surport of On7 2015 G600S

### PR DESCRIPTION
**Pull request by pkj-daeam**
add surport of On7 (2015) G600S

### CODE REVIEW

File: dts/msm8916/msm8916-samsung-r00.dts
Comment (TravMurav) [inline dts/msm8916/msm8916-samsung-r00.dts:18]: We use tabs not spaces for indentation here.

```suggestion
	};

	on7lte-kor {
```

File: dts/msm8916/msm8916-samsung-r00.dts
Comment (TravMurav) [inline dts/msm8916/msm8916-samsung-r00.dts:28]: Let's keep an empty line between nodes.

```suggestion
	};

	coreprimelte {
```

**Review by TravMurav — COMMENTED** [review]
Thanks for adding this! I have few minor nitpicks.
Apart from these, could you please update the readme to add the device to the list?

File: dts/msm8916/msm8916-samsung-r00.dts
Comment (pkj-daeam) [reply dts/msm8916/msm8916-samsung-r00.dts:28]: i dont know how to use github(im newbie) how can i update readme

Comment (TravMurav) [reply dts/msm8916/msm8916-samsung-r00.dts:28]: Just update the readme file. You can look at other commits that add new devices in the commit history :)

Comment (pkj-daeam) [reply dts/msm8916/msm8916-samsung-r00.dts:28]: sorry
i dont know how to add commit on this pull request

Comment (TravMurav) [reply dts/msm8916/msm8916-samsung-r00.dts:28]: I suppose you have already made this change locally and tested that the lk2nd with your changes works on your device. Please make sure these changes look correct (i.e. tabs are used for indentation); make the readme change from here. Create a new branch with the same name as the one used in #260 Then make a new commit (`git add .` then `git commit`, see how other commits write the subject and do it similarly). After that you would need to add your fork as a remote (`git remote add <some name> <your url>`) and update the branch, overwriting it and discarding remote changes (`git push --force`)

Comment (pkj-daeam) [reply dts/msm8916/msm8916-samsung-r00.dts:28]: thanks ill try it

Comment (pkj-daeam) [reply dts/msm8916/msm8916-samsung-r00.dts:28]: pls merge, i edit readme in patch-1

Comment (TravMurav) [reply dts/msm8916/msm8916-samsung-r00.dts:28]: Please note I've asked you to make some changes. I'd also appreciate if you can squash all commits into one and give it a fitting subject. (See other commits in the history, you can use `git rebase --interactive` to squash the changes)

Comment (pkj-daeam) [reply dts/msm8916/msm8916-samsung-r00.dts:28]: Hello? Pls merge

Comment (wonderfulShrineMaidenOfParadise) [reply dts/msm8916/msm8916-samsung-r00.dts:28]: You need to solve all threads before merging.
Please join [#msm8916:postmarketos.org](https://app.element.io/#/room/#msm8916:postmarketos.org) if you have questions.

File: dts/msm8916/msm8916-samsung-r00.dts
Comment (pevik) [inline dts/msm8916/msm8916-samsung-r00.dts:29]: I'd mention in the commit message what this commit really does: adds whitespace to increase readability.

File: dts/msm8916/msm8916-samsung-r00.dts
Comment (pevik) [reply dts/msm8916/msm8916-samsung-r00.dts]: Also here should probably be empty line (you added spaces in this file in previous separate commit and then add new content without blank line?)

File: dts/msm8916/msm8916-samsung-r00.dts
Comment (pevik) [reply dts/msm8916/msm8916-samsung-r00.dts]: > You need to solve all threads before merging.

Or @TravMurav or other maintainer could just amend the change and merge on behalf of the author to speedup.

File: dts/msm8916/msm8916-samsung-r00.dts
Comment (Newbytee) [reply dts/msm8916/msm8916-samsung-r00.dts:29]: I think the commits just should be squashed.

File: dts/msm8916/msm8916-samsung-r00.dts
Comment (Newbytee) [inline dts/msm8916/msm8916-samsung-r00.dts:19]: This line still uses spaces, not tabs.

**Review by Newbytee — APPROVED** [review]
Commits should be squashed and commit message fixed, but I think whoever merges easily can do that.

**Comment by TravMurav** [comment]
Thanks!

**Pull request merged**
